cst438_gradebook/src/test/java/com/cst438/JunitTestGradebook.java

```diff
@@ -258,4 +253,127 @@ private static String asJsonString(final Object obj) {
 		}
 	}
 
+    @Test
+    public void updateAssignmentName() throws Exception {
+        MockHttpServletResponse response;
+
+        Course course = new Course();
+        course.setCourse_id(TEST_COURSE_ID);
+        course.setSemester(TEST_SEMESTER);
+        course.setYear(TEST_YEAR);
+        course.setInstructor(TEST_INSTRUCTOR_EMAIL);
+        course.setEnrollments(new java.util.ArrayList<Enrollment>());
+        course.setAssignments(new java.util.ArrayList<Assignment>());
+
+        Enrollment enrollment = new Enrollment();
+        enrollment.setCourse(course);
+        course.getEnrollments().add(enrollment);
+        enrollment.setId(TEST_COURSE_ID);
+        enrollment.setStudentEmail(TEST_STUDENT_EMAIL);
```

Comment:
Maybe for this you can do a test student email and password?

Reply:
oh, only student name and email are stored on the enrollment